### PR TITLE
Windows movement

### PR DESCRIPTION
**Pull request by gdt050579**

### CODE REVIEW

File: AppCUI/src/Controls/Window.cpp [hunk 1]
```diff
@@ -288,6 +288,39 @@ void WindowRadioButtonClicked(WindowBarItem* start, WindowBarItem* end, WindowBa
...
+void MoveWindowPosTo(Window* win, int addX, int addY, bool keepInDesktopounderies)
+{
+    auto x      = win->GetX() + addX;
+    auto y      = win->GetY() + addY;
+    const int w = win->GetWidth();
+    const int h = win->GetHeight();
+    Size desktopSize;
+    if (AppCUI::Application::GetApplicationSize(desktopSize) == false)
+        return;
+    if (keepInDesktopounderies)
```
Comment (gheorghitamutu) [inline AppCUI/src/Controls/Window.cpp:300]: typo

File: AppCUI/src/Controls/Window.cpp [hunk 2]
```diff
@@ -1072,54 +1112,117 @@ bool Window::OnKeyEvent(Input::Key KeyCode, char16)
...
+            MoveWindowPosTo(this, 0, 1, false);
+            return true;
+        case Key::Left:
+            MoveWindowPosTo(this, -1, 0, false);
+            return true;
+        case Key::Right:
+            MoveWindowPosTo(this, 1, 0, false);
+            return true;
+        case Key::Alt | Key::Up:
+            MoveWindowPosTo(this, 0, -1000000, true);
```
Comment (gheorghitamutu) [inline AppCUI/src/Controls/Window.cpp:1138]: what does -1000000, means?

File: AppCUI/src/Controls/Window.cpp (hunk 1 above)
Comment (gdt050579) [reply AppCUI/src/Controls/Window.cpp]: fixed

File: AppCUI/src/Controls/Window.cpp (hunk 2 above)
Comment (gdt050579) [reply AppCUI/src/Controls/Window.cpp]: fixed

**Review by gheorghitamutu — APPROVED** [review]

**Pull request merged**